src/main/java/com/grupo29/techchallengeriwatts/controller/EnderecoController.java

```diff
@@ -35,8 +33,8 @@ public ResponseEntity registerEndereco(@RequestBody EnderecoDTO enderecoDTO) {
             return ResponseEntity.badRequest().body(violations);
         }
 
-        enderecoRepository.createAddress(enderecoDTO.toEndereco());
-        return ResponseEntity.status(HttpStatus.CREATED).build();
+        Endereco endereco = enderecoRepository.createAddress(enderecoDTO.toEndereco());
+        return new ResponseEntity("Endereço com id %d criado.".formatted(endereco.getId()), HttpStatus.CREATED);
```

Comment:
```suggestion
        Endereco endereco = enderecoRepository.createAddress(enderecoDTO.toEndereco());
        return ResponseEntity.status(HttpStatus.CREATED).body(endereco)
```
Neste caso não seria melhor retornar o objeto criado? Que o response já retornaria o id também.